bundles/org.openhab.binding.bluetooth/README.md

```diff
@@ -37,11 +38,12 @@ Both thing types only require a single configuration parameter `address`, which
 
 ## Channels
 
-Every Bluetooth thing has the following channel:
+Every Bluetooth thing has the following channels:
 
-| Channel ID | Item Type | Description                                                                                         |
-|------------|-----------|-----------------------------------------------------------------------------------------------------|
-| rssi       | Number    | The "Received Signal Strength Indicator", the [RSSI](https://blog.bluetooth.com/proximity-and-rssi) |
+| Channel ID    | Item Type | Description                                                                                         |
+|---------------|-----------|-----------------------------------------------------------------------------------------------------|
+| rssi          | Number    | The "Received Signal Strength Indicator", the [RSSI](https://blog.bluetooth.com/proximity-and-rssi) |
+| last-activity | DateTime  | The last time that any radio activity was received from this device                                 |
```

Comment:
How is that related to this PR?
I am not too keen on such channels - if we feel such information is needed, we should rather solve it for ALL bindings at the same time.
What I could live with here is to keep track of it as a Thing property instead.

Reply:
This PR is a general adaptation of the features available in the 3rd party bluetooth binding. One of the primary features of the binding was presence detection, which a `last-activity` channel would almost certainly be required for. 

While beacon bluetooth devices could be tracked by a change in their RSSI channel, such a method can't be used for connected devices which don't send out advertisement information that would trigger an RSSI channel update. (This is the case for devices tracked through Bluez adapters, but not for Bluegiga, because they are special I guess).

The "last-activity" channel would be updated on either rssi changes or on successful responses during connected communications.

I can change this to a thing property if you wish, but I don't know if users will be able to make rules that would use it then. WDYT?

Reply:
Can you elaborate what kind of rules people write when having that channel available? Could the logic of such rules possibly be captured within the binding and be provided as a special channel?

For presence detection, I would possibly consider the Thing status to be the best instrument - if the Thing is OFFLINE, it means the device isn't in reach anymore. BT things could have a configuration parameter, which says after what period of inactivity the Thing should move to OFFLINE status. Would that address the requirement?

Reply:
> For presence detection, I would possibly consider the Thing status to be the best instrument - if the Thing is OFFLINE, it means the device isn't in reach anymore. BT things could have a configuration parameter, which says after what period of inactivity the Thing should move to OFFLINE status. Would that address the requirement?

You have a point there. I'll adjust the feature to control thing status and get rid of the `last-activity` channel. Thank for pointing this out.